src/Api/Command/PayByBlik.php

```diff
@@ -8,7 +8,9 @@ final class PayByBlik
 {
     public function __construct(
         public readonly int $paymentId,
-        public readonly string $blikToken,
+        public readonly ?string $blikToken,
+        public readonly bool $blikSaveAlias = false,
+        public readonly bool $blikUseAlias = false,
```

Comment:
don't we need to send the selected alias anyway? perhaps instead of two fields `blikSaveAlias/blikUseAlias` we can define something like `operationType` with some enum behind

Reply:
We don't need to send the alias, as there is only one alias per customer. `operationType` is indeed a good idea. I will do that after I finish the rest of the features in this PR.

Reply:
fixed